cli/CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - `transaction` and `transactions` indexed transactions in opposite order;
+- Enter doesn't show helper anymore;
```

Comment:
I thought we don't like using temporal terms like `now`, `anymore`, etc

Also ... I have a question ... Under the "Fixed" tab, should we describe what was wrong before like I did in line 36, or should we put a description of the new fixed state there?

Reply:
As long as it's understandable, I'd say it doesn't matter. I'll reword.

Reply:
Done